### PR TITLE
Adjusting package dependencies

### PR DESCRIPTION
**Pull request by denisonbarbosa**
Moving curlftpfs package from a dependency to a recommendation, as it's on universe rather than main.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -35,9 +35,9 @@ Depends: ${shlibs:Depends},
          sssd-dbus,
          apparmor,
          cifs-utils,
-         curlftpfs,
 Recommends: ${misc:Recommends},
             ubuntu-advantage-desktop-daemon,
+            curlftpfs,
```
Comment (didrocks) [inline debian/control:40]: This is a Recommends, not a Suggests. As we discussed duringour HO, Ubuntu installs recommends by default, but not Suggests.

File: doc/09.-Network-shares.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ The mount process for these mounts is triggered at the moment a client logs in.
 
 All protocols supported by the [mount command](https://manpages.ubuntu.com/manpages/jammy/en/man8/mount.8.html) should work out of the box. However, the only tested ones are `smb`, `ftp` and `nfs`.
 
+The backends for the protocols `smb` and `nfs` are automatically enabled when installing the adsys package. In order to enable the backend for `ftp` mounts, the user must install the recommended `curlftpfs` package.
```
Comment (didrocks) [inline doc/09.-Network-shares.md:15]: Maybe mentions that this is valid on Ubuntu? (in case some other distros carries adsys one day)

**Review by didrocks — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (denisonbarbosa) [reply debian/control:40]: Oh, I switched the definitions mentally... My bad, brain is still a bit slow after the holidays

**Review by didrocks — APPROVED** [review]
let’s goooooooooo

**Pull request merged**